main.js

```diff
@@ -0,0 +1,43 @@
+const userData = []
+
+const showUsers = function () {
+
+    $.ajax({
```

Comment:
You should separate file and concepts, especially when using MVC, write your render functions in a different place